static/index.html

```diff
@@ -35,6 +35,15 @@
                     <a href="https://github.com/mattgodbolt/compiler-explorer">
                         <img height="20" width="20" src="assets/GitHub-Mark-20px.png">&nbsp;Source on GitHub</a>
                 </li>
+                <li class="dropdown">
+                    <a href="#" class="dropdown-toggle" data-toggle="dropdown">Other Languages<b class="caret"></b></a>
+                    <ul class="dropdown-menu">
+                        <li><a href="http://gcc.beta.godbolt.org">C++</a></li>  
```

Comment:
Also, note that https://gcc.beta.godbolt.org/ throws a "connection refused" error